File: src/main/java/com/gmail/borlandlp/minigamesdtools/arena/localevent/ArenaPlayerJoinLocalEvent.java
package com.gmail.borlandlp.minigamesdtools.arena.localevent;

import com.gmail.borlandlp.minigamesdtools.arena.team.TeamProvider;
import org.bukkit.entity.Player;

@ArenaEventHandler
public class ArenaPlayerJoinLocalEvent extends ArenaEvent {
    private Player player;
    private TeamProvider team;

    public ArenaPlayerJoinLocalEvent(Player player, TeamProvider team) {
        this.player = player;
        this.team = team;
    }

    public Player getPlayer() {
        return player;
    }

    public TeamProvider getTeam() {
        return team;
    }

    public String toString() {
        return "{Event:" + this.getClass().getSimpleName() + " Player" + this.getPlayer() + " TeamProvider:" + this.getTeam() +"}";
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/arena/localevent/ArenaPlayerJoinLocalEvent.kt
package com.gmail.borlandlp.minigamesdtools.arena.localevent

import com.gmail.borlandlp.minigamesdtools.arena.team.TeamProvider
import org.bukkit.entity.Player

@ArenaEventHandler
class ArenaPlayerJoinLocalEvent(val player: Player, val team: TeamProvider) : ArenaEvent() {
    override fun toString(): String {
        return "{Event:" + this.javaClass.simpleName + " Player" + player + " TeamProvider:" + team + "}"
    }
}